services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -237,22 +243,34 @@ export const DashboardPage = ({ reportType }: Props) => {
     <PageTemplate type="report" sx={sx.layout}>
       <Link as={RouterLink} to="/" sx={sx.returnLink}>
         <Image src={arrowLeftIcon} alt="Arrow left" className="returnIcon" />
-        Return Home
+        Return home
       </Link>
       {errorMessage && <ErrorAlert error={errorMessage} />}
+      {/* Only show SAR alert banner if the corresponding Work Plan is not approved */}
+      {reportType === ReportType.SAR &&
+        workPlanStatus !== ReportStatus.APPROVED && (
+          <Alert
```

Comment:
talked to Kim and we should be left-aligning this alert banner to the page content
![Uploading Screenshot 2023-10-16 at 3.08.33 PM.png…]()

Reply:
just seeing britt's comment :) 
I bet centering is preferred and what we've done in the past

Reply:
I just updated that following the Figma, let me ask Kim!